.github/workflows/platform-install-postgres.yml

```diff
@@ -3,7 +3,7 @@ on:
   push:
     branches: [ master ]
   repository_dispatch:
-    types: [qa-initialSetup]
+    types: [qa]
```

Comment:
ensure it's only your changes that are being reflected?

Reply:
Thanks. i resolved that.